lib/BaseCalendarApiAdapter.ts

```diff
@@ -304,10 +306,10 @@ export class BaseCalendarApiAdapter {
             vcalendar.getFirstSubcomponent("vtimezone")?.getFirstPropertyValue("tzid") || "";
 
           const startDate = calendarTimezone
-            ? dayjs(event.startDate.toJSDate()).tz(calendarTimezone)
+            ? dayjs.tz(event.startDate.toString(), calendarTimezone)
```

Comment:
I don't think you need toString() here

Reply:
without toString ts will complain
<img width="833" alt="Screen Shot 2021-12-27 at 8 19 45 PM" src="https://user-images.githubusercontent.com/9290574/147471281-1c2b6a92-68b4-48d2-b00d-19bb2d524cd0.png">